README.md

```diff
@@ -42,29 +57,23 @@ You can find the Clay documentation on the [site](https://clayui.com/docs).
 
 You can improve by sending pull requests to this repository.
 
-## Installation
-
-Clay is available as the `clay` package in npm for components and for css like `clay-css`. It is also available on a [CDN](https://cdn.jsdelivr.net/npm/clay/lib/js/clay.js) for components and for css, [check](https://cdn.jsdelivr.net/npm/clay/lib/css/atlas.css). But if you just want the separate components, it's also available in the npm as [clay-](https://www.npmjs.com/search?q=clay-).
-
-See the [Getting Started](https://clayui.com/docs/get-started/index.html) guide for more information.
-
 ## Migration Guides
 
 Do you already have the Clay v2 components implemented in your project? These handy guides will help you perform the migration from [Clay v2](https://v2.clayui.com) to [v3](https://clayui.com).
 
 -   [Migrate the Clay components from v2 to v3](https://clayui.com/docs/get-started/migrate-the-clay-components-from-v2-to-v3.html)
 
-## Contributing
+## Installation
 
-Feel free to open up problems or send pull requests. We will always be looking at these problems and we will be responding whenever possible.
+Clay is available as the `clay` package in npm for components and for css like `clay-css`. It is also available on a [CDN](https://cdn.jsdelivr.net/npm/clay/lib/js/clay.js) for components and for css, [check](https://cdn.jsdelivr.net/npm/clay/lib/css/atlas.css). But if you just want the separate components, it's also available in the npm as [clay-](https://www.npmjs.com/search?q=clay-).
 
-> Before opening a issue make sure it exists.
+See the [Getting Started](https://clayui.com/docs/get-started/index.html) guide for more information.
 
-See the [contribution guide](/CONTRIBUTING.md) for more details.
+## Setup
 
-### Setup
+-   #### **Packages**
 
-#### Packages
+To contribute to Clay React Components, you can try to run locally to test your changes:
 
 1. Install NodeJS >= [v6.11.0](http://nodejs.org/dist/v6.11.0/), if you don't have it yet.
```

Comment:
I would suggest, installing the latest "LTS" version, that way we don't need to update this every 6 weeks

Reply:
I'll change the Node version to 16.x, it seems that we are updated 😄

Reply:
@sergiojimcos, I was actually saying that you shouldn't mention any specific version: clay should work with whatever LTS version is available. Don't worry I've fetched the branch and will push more stuff on top of your changes.